README.md

```diff
@@ -681,6 +667,10 @@ Peer Reviewing by looking at PR code changes is nice.
 
 Testing PR code changes on real sites is extra beneficial.
 
+### Local Development
```

Comment:
A new heading, you'll need to run the command to update the table of contents in the file. We should really have a GitHub workflow that fails if this hasn't been updated.

Reply:
It would be kind of neat if github could rerun doctoc and then push a new commit. I can imagine other tools where we'd want to do something like that too.

Reply:
I updated the toc. Looks like #567 covers the suggested improvements.